Bottom: a41641818a4e65e9584618c07f96aabefd394ad2
Top:    aec1fa5d5dcb923834e84ac09b863845595d15c6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:07 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 85bffb4..37340d4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2707,7 +2707,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
